src/Text/Pandoc/Writers/Txt2Tags.hs

```diff
@@ -20,8 +20,7 @@ Txt2Tags:  <https://www.Txt2Tags.org/>
    - Definition lists are broken
    - Tables could be improved
    - Some formats make better results than others (html is ok, md is somehow broken)
-   - code related to dokuwiki only could be removed
```

Comment:
Is there still dokuwiki-specific code?  IF so, let's remove it.

Reply:
I suppose there are still some code, yes